app/controllers/support_interface/performance_controller.rb

```diff
@@ -18,6 +18,13 @@ def submitted_application_choices
       send_data csv, filename: "submitted-application-choices-#{Time.zone.today}.csv", disposition: :attachment
     end
 
+    def candidate_journey_tracking
```

Comment:
If we add this then we can delete the `application_timings` (ApplicationChoicesExport) export, it serves the same purpose.

Reply:
👍 I'd like to do this as a follow-up PR. This one has already dragged on for too long.

Reply:
@tijmenb Mike has asked us to keep the old report - he needs it for the short-term at least.